### PR TITLE
Fix unbound local error exception

### LINKED ISSUE #2424: Error when posting a xml submission file
### Environmental Information

- Onadata version: 3.9.0

### Problem description

The error `500 Internal Server Error` is thrown when posting an xml_submission_file on both staging and production environments.

### Expected behavior

A user should be able to post an xml_submission_file to a form on both staging and production environments as long as the xml file is correctly formatted and the related form exists in the account.

### Steps to reproduce the behavior

- POST an xml_submission_file to both staging and production environments.

### Additional Information

Screenshot from 2023-05-16 12-36-46

_Logs, [related issues](github.com/onaio/onadata/issues), weird / out of place occurrences, local settings, possible approach to solving this..._

### PR DESCRIPTION
**Pull request by KipSigei**
### Changes / Features implemented
- Add postman to list of user agents in the analytics module
- Fix `UnboundLocalError` exception thrown for uninitialized `event_source` var
### Side effects of implementing this change

**Before submitting this PR for review, please make sure you have:**

  - [x] Included tests
  - [ ] Updated documentation

Closes #2424 and #2427

### CODE REVIEW

**Comment by DavisRayM** [comment]
Let's have this component disabled by default too

**Review by ukanga — COMMENTED** [review]
@KipSigei Issue #909 applies to the export_async endpoint. How does this PR address the same issue yet the paths are different?

**Comment by KipSigei** [comment]
> @KipSigei Issue #909 applies to the export_async endpoint. How does this PR address the same issue yet the paths are different?

@ukanga I meant to tag this issue https://github.com/onaio/onadata/pull/2428#issuecomment-1558819927

**Review by DavisRayM — DISMISSED** [review]

File: onadata/libs/utils/analytics.py [hunk 1]
```diff
@@ -136,18 +136,18 @@ def get_event_label(self) -> str:
     def get_request_origin(self, request, tracking_properties):
         """Returns the request origin"""
         if isinstance(self.tracked_obj, Instance):
+            event_source = ""  # Initialize event_source variable
```
[the remark this answers is not in the document]
Comment (KipSigei) [reply onadata/libs/utils/analytics.py]: @ukanga Addressed via https://github.com/onaio/onadata/pull/2428/commits/33ea555d9c1d4b0e55866fb4066dd157d3a4de5f

File: onadata/libs/utils/analytics.py [hunk 2]
```diff
@@ -135,21 +135,21 @@ def get_event_label(self) -> str:
...
-                    or "Safari" in user_agent
-                ):
-                    event_source = "Submission collected from Enketo"
             except KeyError:
-                event_source = ""
-        else:
-            event_source = ""
+                pass
+
+            if "Android" in user_agent:
```
Comment (DavisRayM) [inline onadata/libs/utils/analytics.py:148]: I believe `user_agent` won't be defined in scenarios where the KeyError is raised

**Review by DavisRayM — APPROVED** [review]

**Pull request merged**